### PR TITLE
CTD & METBK cals

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYV Instrument Cal CSVs [DT-184]](https://trello.com/c/Eo546k32/184-pyv-instrument-cal-csvs-dt-184)

**Review by sauuyer — APPROVED** [review]
All compared agains .cal files in alfresco - all look good. No revisions.

File: calibration/CTDBPF/CGINS-CTDBPF-50116__20230311.csv [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+16-50116,CC_pa2,1.026143e-011,
+16-50116,CC_ptca0,5.099838e+005,
+16-50116,CC_ptca1,-2.807320e+002,
+16-50116,CC_ptca2,-1.191795e+000,
+16-50116,CC_ptcb0,2.508125e+001,
+16-50116,CC_ptcb1,2.500000e-004,
+16-50116,CC_ptcb2,0.000000e+000,
+16-50116,CC_ptempa0,4.072507e+001,
+16-50116,CC_ptempa1,-4.366180e+001,
+16-50116,CC_ptempa2,-4.798972e-001,
```
Comment (sbeaulieu) [inline calibration/CTDBPF/CGINS-CTDBPF-50116__20230311.csv:23]: looks good

File: calibration/CTDBPP/CGINS-CTDBPP-50126__20230322.csv [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+16-50126,CC_pa2,9.182062e-011,
+16-50126,CC_ptca0,5.261769e+005,
+16-50126,CC_ptca1,2.691677e+001,
+16-50126,CC_ptca2,6.033252e-001,
+16-50126,CC_ptcb0,2.504025e+001,
+16-50126,CC_ptcb1,-3.500000e-004,
+16-50126,CC_ptcb2,0.000000e+000,
+16-50126,CC_ptempa0,3.963294e+002,
+16-50126,CC_ptempa1,-2.394848e+002,
+16-50126,CC_ptempa2,3.505068e+001,
```
Comment (sbeaulieu) [inline calibration/CTDBPP/CGINS-CTDBPP-50126__20230322.csv:23]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-12305__20230321.csv [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+37-12305,CC_ptca0,5.249661e+005,
+37-12305,CC_ptca1,4.127303e+000,
+37-12305,CC_ptca2,-3.426805e-002,
+37-12305,CC_ptcb0,2.549325e+001,
+37-12305,CC_ptcb1,-5.500000e-004,
+37-12305,CC_ptcb2,0.000000e+000,
+37-12305,CC_ptempa0,-6.845244e+001,
+37-12305,CC_ptempa1,5.153888e-002,
+37-12305,CC_ptempa2,-5.100431e-007,
+37-12305,CC_p_range,1450,
```
Comment (sbeaulieu) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12305__20230321.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-12307__20230315.csv [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+37-12307,CC_ptca0,5.242415e+005,
+37-12307,CC_ptca1,5.513499e+000,
+37-12307,CC_ptca2,-1.272956e-001,
+37-12307,CC_ptcb0,2.535075e+001,
+37-12307,CC_ptcb1,-2.850000e-003,
+37-12307,CC_ptcb2,0.000000e+000,
+37-12307,CC_ptempa0,-7.390094e+001,
+37-12307,CC_ptempa1,5.095836e-002,
+37-12307,CC_ptempa2,-6.345900e-007,
+37-12307,CC_p_range,1450,
```
Comment (sbeaulieu) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12307__20230315.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-12607__20230321.csv [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+37-12607,CC_ptca0,5.248304e+005,
+37-12607,CC_ptca1,-7.452031e-002,
+37-12607,CC_ptca2,-2.173538e-002,
+37-12607,CC_ptcb0,2.554038e+001,
+37-12607,CC_ptcb1,2.750000e-004,
+37-12607,CC_ptcb2,0.000000e+000,
+37-12607,CC_ptempa0,-6.591847e+001,
+37-12607,CC_ptempa1,5.151877e-002,
+37-12607,CC_ptempa2,-4.155579e-007,
+37-12607,CC_p_range,1450,
```
Comment (sbeaulieu) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12607__20230321.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-12635__20230330.csv [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+37-12635,CC_ptca0,5.245101e+005,
+37-12635,CC_ptca1,1.837833e+000,
+37-12635,CC_ptca2,2.193718e-002,
+37-12635,CC_ptcb0,2.549575e+001,
+37-12635,CC_ptcb1,-2.500000e-004,
+37-12635,CC_ptcb2,0.000000e+000,
+37-12635,CC_ptempa0,-7.129122e+001,
+37-12635,CC_ptempa1,4.958098e-002,
+37-12635,CC_ptempa2,-2.228129e-007,
+37-12635,CC_p_range,1450,
```
Comment (sbeaulieu) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12635__20230330.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-12636__20230315.csv [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+37-12636,CC_ptca0,5.245294e+005,
+37-12636,CC_ptca1,5.936901e+000,
+37-12636,CC_ptca2,-5.680366e-002,
+37-12636,CC_ptcb0,2.550763e+001,
+37-12636,CC_ptcb1,3.250000e-004,
+37-12636,CC_ptcb2,0.000000e+000,
+37-12636,CC_ptempa0,-6.774060e+001,
+37-12636,CC_ptempa1,5.137560e-002,
+37-12636,CC_ptempa2,-5.031340e-007,
+37-12636,CC_p_range,1450,
```
Comment (sbeaulieu) [inline calibration/CTDMOQ/CGINS-CTDMOQ-12636__20230315.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-13379__20230321.csv [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+37-13379,CC_ptca0,5.241628e+005,
+37-13379,CC_ptca1,8.750851e-001,
+37-13379,CC_ptca2,2.288397e-003,
+37-13379,CC_ptcb0,2.505987e+001,
+37-13379,CC_ptcb1,-2.500000e-005,
+37-13379,CC_ptcb2,0.000000e+000,
+37-13379,CC_ptempa0,1.442079e+002,
+37-13379,CC_ptempa1,-6.965312e-002,
+37-13379,CC_ptempa2,1.062329e-006,
+37-13379,CC_p_range,1450,
```
Comment (sbeaulieu) [inline calibration/CTDMOQ/CGINS-CTDMOQ-13379__20230321.csv:25]: looks good

File: calibration/CTDMOQ/CGINS-CTDMOQ-13380__20230321.csv [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+37-13380,CC_ptca0,5.250121e+005,
+37-13380,CC_ptca1,5.670362e-001,
+37-13380,CC_ptca2,3.063318e-001,
+37-13380,CC_ptcb0,2.503225e+001,
+37-13380,CC_ptcb1,5.000000e-005,
+37-13380,CC_ptcb2,0.000000e+000,
+37-13380,CC_ptempa0,1.565064e+002,
+37-13380,CC_ptempa1,-6.877314e-002,
+37-13380,CC_ptempa2,6.932930e-007,
+37-13380,CC_p_range,1450,
```
Comment (sbeaulieu) [inline calibration/CTDMOQ/CGINS-CTDMOQ-13380__20230321.csv:25]: looks good

File: calibration/CTDMOR/CGINS-CTDMOR-13375__20230314.csv [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+37-13375,CC_ptca0,5.227645e+005,
+37-13375,CC_ptca1,4.516810e-001,
+37-13375,CC_ptca2,-2.925703e-002,
+37-13375,CC_ptcb0,2.497814e+001,
+37-13375,CC_ptcb1,-1.371571e-003,
+37-13375,CC_ptcb2,0.000000e+000,
+37-13375,CC_ptempa0,-6.926935e+001,
+37-13375,CC_ptempa1,5.062971e-002,
+37-13375,CC_ptempa2,-4.383899e-007,
+37-13375,CC_p_range,5076,
```
Comment (sbeaulieu) [inline calibration/CTDMOR/CGINS-CTDMOR-13375__20230314.csv:25]: looks good

File: calibration/CTDMOR/CGINS-CTDMOR-13376__20230314.csv [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+37-13376,CC_ptca0,5.246469e+005,
+37-13376,CC_ptca1,2.639845e+000,
+37-13376,CC_ptca2,-6.838049e-002,
+37-13376,CC_ptcb0,2.500613e+001,
+37-13376,CC_ptcb1,-1.750000e-004,
+37-13376,CC_ptcb2,0.000000e+000,
+37-13376,CC_ptempa0,2.559695e+002,
+37-13376,CC_ptempa1,-1.162566e-001,
+37-13376,CC_ptempa2,2.929281e-006,
+37-13376,CC_p_range,5076,
```
Comment (sbeaulieu) [inline calibration/CTDMOR/CGINS-CTDMOR-13376__20230314.csv:25]: looks good

File: calibration/CTDMOR/CGINS-CTDMOR-13377__20230517.csv [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+37-13377,CC_ptca0,5.246697e+005,
+37-13377,CC_ptca1,1.687245e+001,
+37-13377,CC_ptca2,-9.146414e-002,
+37-13377,CC_ptcb0,2.498750e+001,
+37-13377,CC_ptcb1,-3.000000e-004,
+37-13377,CC_ptcb2,0.000000e+000,
+37-13377,CC_ptempa0,-7.013064e+001,
+37-13377,CC_ptempa1,5.086466e-002,
+37-13377,CC_ptempa2,-4.631382e-007,
+37-13377,CC_p_range,5076,
```
Comment (sbeaulieu) [inline calibration/CTDMOR/CGINS-CTDMOR-13377__20230517.csv:25]: looks good

File: calibration/CTDMOR/CGINS-CTDMOR-13654__20230314.csv [hunk 13]
```diff
@@ -0,0 +1,25 @@
...
+37-13654,CC_ptca0,5.241639e+005,
+37-13654,CC_ptca1,2.220491e-001,
+37-13654,CC_ptca2,1.006615e-002,
+37-13654,CC_ptcb0,2.520675e+001,
+37-13654,CC_ptcb1,-6.500000e-004,
+37-13654,CC_ptcb2,0.000000e+000,
+37-13654,CC_ptempa0,1.999167e+002,
+37-13654,CC_ptempa1,-9.482824e-002,
+37-13654,CC_ptempa2,1.530057e-006,
+37-13654,CC_p_range,5076,
```
Comment (sbeaulieu) [inline calibration/CTDMOR/CGINS-CTDMOR-13654__20230314.csv:25]: looks good

File: calibration/METBKA/CGINS-METCTD-11090__20230423.csv [hunk 14]
```diff
@@ -0,0 +1,12 @@
...
+37-11090,CC_a1,3.131041e-004,
+37-11090,CC_a2,-5.056132e-006,
+37-11090,CC_a3,2.164972e-007,
+37-11090,CC_g,-1.014477e+000,
+37-11090,CC_h,1.443887e-001,
+37-11090,CC_i,-1.695599e-004,
+37-11090,CC_j,3.413202e-005,
+37-11090,CC_ctcor,3.250000e-006,
+37-11090,CC_cpcor,-9.570000e-008,
+37-11090,CC_wbotc,2.827399e-007,
```
Comment (sbeaulieu) [inline calibration/METBKA/CGINS-METCTD-11090__20230423.csv:12]: looks good

File: calibration/METBKA/CGINS-METCTD-12755__20230311.csv [hunk 15]
```diff
@@ -0,0 +1,12 @@
...
+37-12755,CC_a1,3.047704e-004,
+37-12755,CC_a2,-4.295382e-006,
+37-12755,CC_a3,1.964261e-007,
+37-12755,CC_g,-9.924316e-001,
+37-12755,CC_h,1.536068e-001,
+37-12755,CC_i,-3.347892e-005,
+37-12755,CC_j,2.739397e-005,
+37-12755,CC_ctcor,3.250000e-006,
+37-12755,CC_cpcor,-9.570000e-008,
+37-12755,CC_wbotc,9.837923e-008,
```
Comment (sbeaulieu) [inline calibration/METBKA/CGINS-METCTD-12755__20230311.csv:12]: looks good

**Review by sbeaulieu — APPROVED** [review]

**Pull request merged**